ksqldb-serde/src/main/java/io/confluent/ksql/serde/delimited/KsqlDelimitedSerializer.java

```diff
@@ -92,13 +93,20 @@ public Object next() {
       switch (column.type().baseType()) {
         case DECIMAL:
           return handleDecimal((BigDecimal) value);
+        case TIME:
+          return handleTime((Time) value);
         case TIMESTAMP:
           return handleTimestamp((Timestamp) value);
         default:
           return value;
       }
     }
 
+    private static Integer handleTime(final Time value) {
+      // Return milliseconds
+      return value == null ? null : (int) value.getTime();
```

Comment:
Why isn't using `long`? I see that `getTime()` returns long. In fact, I just noticed that we are using INT32 for times and you're returning `longValue()` in the connector converter. Is there a problem with using long and int in different contexts?

Reply:
Avro and Protobuf store Time as int, so we're serializing everything to int for consistency. The longValue() in the connect converter eventually gets converted to a `java.sql.Time` object, so it doesn't cause problems, but I've updated that and others to use int so that its clearer.